packages/core/strapi/lib/utils/get-dirs.js

```diff
@@ -11,7 +11,7 @@ const getDirs = root => ({
   policies: join(root, 'src', 'policies'),
   middlewares: join(root, 'src', 'middlewares'),
   config: join(root, 'config'),
-  public: join(root, 'public'),
+  public: resolve(root, strapi.config.get('server.dirs.public')),
```

Comment:
shouldn't we keep the 'public' default too?

Reply:
What do you mean?
The default is set here : https://github.com/strapi/strapi/pull/12534/files#diff-39218746791a552985288c12df91fd73aa563b8308dbdc502702a63d0659113aR24